common/k8s/src/middleware/metadata.rs

```diff
@@ -100,13 +100,41 @@ impl K8sMetadata {
                 let obj_ref = ObjectRef::from_obj(&pod);
                 if reader.get(&obj_ref).is_none() {
                     Metrics::k8s().increment_creates();
+                    info!(
```

Comment:
info might be too chatty. agent sends these logs to backend too. affects the bill.

Reply:
I modified the code to track generic pod changes with the info level, and more detailed annotation/label information with the trace level.

I thought the metadata cache tracking might be good to track in general. Let me know if you think I should just make it all trace.